app/imports/templates/components/bid.js

```diff
@@ -16,23 +16,26 @@ Template['components_bid'].onCreated(function() {
   let template = Template.instance();
   let bid = template.data.bid;
   updateRevealedStatus(template, bid);
+  TemplateVar.set(template, 'revealing', false)
 })
 
 Template['components_bid'].events({
-  'click .reveal-bid': function() {
-    let template = Template.instance();
+  'click .reveal-bid': function(e, template) {
+    if (web3.eth.accounts.length == 0) {
+      GlobalNotification.error({
+          content: 'No accounts added to dapp',
+          duration: 3
+      });
+      return;
+    }
     let bid = template.data.bid.bid ? template.data.bid.bid : template.data.bid;
-    MyBids.update({ _id: bid._id }, { $set: {revealing: true} });
-    
+    TemplateVar.set(template, 'revealing', true);
```

Comment:
add the name to the template variable name, `'revealing-'+name`. This way, if the user clicks on another name while it waits for it to be revealed, the next page won't be in the same state